tests/testthat/test-rmot_models_canham.R

```diff
@@ -0,0 +1,38 @@
+#Testing for canham model
+test_that("Model structures: canham", {
+  # Single individual
+  expect_named(rmot_model("canham_single_ind"))
```

Comment:
this should have actual names for comparison. 

e.g. if we have `z <- list(a=1, b=2)` then we'd run `expect_named(x, c("a", "b"))`

Also, why not have 
x <-rmot_model("canham_single_ind")
expect_named(x)
expect_type(x, "list")
expect_visible(x)

Reply:
Done.